src/lib.rs

```diff
@@ -68,7 +68,7 @@
 //! `()` for `Ext`, unless specifically using an inspector or a customized EVM.
 //!
 //! To help you use concrete types, we provide the [`trevm_aliases`] macro. This
-//! macro generates type aliases for the Trevm states with a concrete `Ext` and
+//! macro generates type aliases for the Trevm states with a concrete `Ext` and `Db` types.
```

Comment:
@prestwich this is the only content change, i took a stab at finishing the sentence. is this accurate?

Reply:
yes, it's accurate. grammar nit tho, need to remove the `a` from `a concrete` now